misk-crypto/src/main/kotlin/misk/crypto/KeyManager.kt

```diff
@@ -0,0 +1,18 @@
+package misk.crypto
+
+/**
+ * Singleton object used as a map of key name to its corresponding Cipher implementation.
+ * It's being populated by the {@link CryptoModule} class when loaded.
+ */
+object KeyManager {
+
+  private var keys: HashMap<String, Cipher> = HashMap()
```

Comment:
https://publicobject.com/2016/02/08/linkedhashmap-is-always-better-than-hashmap/

Reply:
or `mutableMapOf<String, Cipher>()`

Reply:
👍